sentry/src/main/java/io/sentry/TransportFactory.java

```diff
@@ -0,0 +1,9 @@
+package io.sentry;
+
+import io.sentry.transport.ITransport;
+import org.jetbrains.annotations.NotNull;
+
+/** Creates instances of {@link ITransport}. */
+public interface TransportFactory {
```

Comment:
should we do `ITransportFactory`?

Reply:
I have mixed feelings about putting `I` in the front of names for interfaces, but I will just follow your preference. Tell me :-)

Reply:
no feelings or preferences here, but see `ITransport`, so ideally, we do a single way, either all of them or none of them.
chose which ones to rename :)

Reply:
Let it be `ITransportFactory`.